admin/services/data-access/pupil-register-v2.data.service.js

```diff
@@ -0,0 +1,34 @@
+'use strict'
+
+const { TYPES } = require('./sql.service')
+
+const sqlService = require('./sql.service')
+
+const service = {
+  getPupilRegister: async function (schoolId) {
```

Comment:
please add JSDoc comments

Reply:
also - you should be able to remove the async and just document the return value as a promise, as you don't await anything

Reply:
thanks for that, added.